src/Console/ConsoleIo.php

```diff
@@ -266,7 +266,7 @@ public function ask($prompt, $default = null)
      */
     public function outputAs($mode)
```

Comment:
@markstory Just to clearify this. I deprecated `ConsoleOutput::outputAs`. Should I also deprecate `ConsoleIo::outputAs `? `ConsoleIo::outputAs` currently acts as a setter only.